alerter.py

```diff
@@ -5,6 +5,9 @@ def network_alert_stub(celcius):
     # Return 200 for ok
     # Return 500 for not-ok
     # stub always succeeds and returns 200
+    if(celcius > 200):
+        return 500
+        
     return 200
 
 def alert_in_celcius(farenheit):

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.

The production code should not be aware of test code or test stub. It should be abstracted by the use of interface or function pointer.